sdv/constraints/tabular.py

```diff
@@ -899,21 +970,16 @@ def reverse_transform(self, table_data):
                 Transformed data.
         """
         table_data = table_data.copy()
-        low = self._get_low_value(table_data)
-        high = self._get_high_value(table_data)
-        data = table_data[self._transformed_column]
+        data = table_data[self.column_name]
 
         data = 1 / (1 + np.exp(-data))
-        data = (data - 0.025) / 0.95
-        data = data * (high - low) + low
-        data = data.clip(low, high)
+        data = data * (self.high_value - self.low_value) + self.low_value
+        data = data.clip(self.low_value, self.high_value)
```

Comment:
Perhaps we should make util functions called `sigmoid` and `logit` that do these operations. That way the code doesn't get reused in two places and we provide some context to the operations. We can also handle the precision problem in those functions